source/algorithms.tex

```diff
@@ -4057,7 +4057,7 @@
 \pnum
 \returns \tcode{true} if
 \range{first}{last} is empty or if
-\range{first}{last} is partitioned by \tcode{pred}, i.e. if all elements that satisfy \tcode{pred} appear before those that do not.
+\range{first}{last} is partitioned by \tcode{pred}, i.e., if all elements that satisfy \tcode{pred} appear before those that do not.
```

Comment:
Personally, I believe the "comma after i.e." thing is totally misguided. Nobody knows what "i.e." and "e.g." means in the expanded form, plus it's Latin anyway, so the argument "oh, it's a complete sentence lookalike, so should have commas around it" is broken. (Have you ever seen the expanded form in English?). We don't put a comma after "but" or "where" either.